contracts/PolygonZkEVM.sol

```diff
@@ -427,18 +438,24 @@ contract PolygonZkEVM is OwnableUpgradeable, EmergencyManager {
         bytes32 currentAccInputHash = sequencedBatches[currentBatchSequenced]
             .accInputHash;
 
+        // Store in a temporal variable, for avoid access again the storage slot
+        uint64 orgLastForceBatchSequenced = currentLastForceBatchSequenced;
```

Comment:
Why `orgLastForceBatchSequenced` naming ?
I think it is better `originLastForceBatchSequenced` or `init...` for readability

```suggestion
        uint64 orgLastForceBatchSequenced = currentLastForceBatchSequenced;
```

Reply:
resolved in future following PR ( Since is not an important issue i will pospone it to avoid rebases)